app/code/Codifi/CustomerRequest/Controller/Request/Save.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Codifi_CustomerRequest
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\CustomerRequest\Controller\Request;
+
+use Magento\Framework\App\Action\Action;
+use Codifi\Training\Model\CustomerNoteFactory;
+use Codifi\Training\Model\NoteRepository;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Framework\Escaper;
+use Magento\Framework\Message\ManagerInterface;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Exception\MailException;
+use Magento\Framework\App\Area;
+use Magento\Store\Model\Store;
+use Magento\Framework\Controller\ResultFactory;
+
+/**
+ * Class Save
+ * @package Codifi\CustomerRequest\Controller\Request
+ */
+class Save extends Action
+{
+    /**
+     * Customer note factory
+     *
+     * @var CustomerNoteFactory
+     */
+    private $customerNoteFactory;
+
+    /**
+     * Customer note repository
+     *
+     * @var NoteRepository
+     */
+    private $customerNoteRepository;
+
+    /**
+     * Transport builder
+     *
+     * @var TransportBuilder
+     */
+    private $transportBuilder;
+
+    /**
+     * Escaper
+     *
+     * @var Escaper
+     */
+    private $escaper;
+
+    /**
+     * Store manager interface
+     *
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
+
+    /**
+     * Message manager
+     *
+     * @var ManagerInterface
+     */
+    private $_messageManager;
```

Comment:
1. According to PSR-12: Property names MUST NOT be prefixed with a single underscore to indicate protected or private visibility.
2. MessageManager is already injected in parent class